utils/types.go

```diff
@@ -156,10 +131,15 @@ func GetKongClient(opt KongClientConfig) (*kong.Client, error) {
 	defaultTransport.TLSClientConfig = &tlsConfig
 	c.Transport = defaultTransport
 	if len(opt.Headers) > 0 {
-		c.Transport = &HeaderRoundTripper{
-			headers: opt.Headers,
-			rt:      defaultTransport,
+
+		headers := http.Header{}
```

Comment:
Please put this in a `func parseHeaders(raw []string) http.Header` and unit test that function.

Reply:
If we decide to implement the suggestion from https://github.com/Kong/deck/pull/338#discussion_r619089783, the return value will probably become `(http.Header, error)`